.github/pull_request_template.md

```diff
@@ -18,4 +18,5 @@
 - [ ] Tests for the changes have been added / updated.
 - [ ] Documentation comments have been added / updated.
 - [ ] Code has been formatted via `zk fmt` and `zk lint`.
-- [ ] Spellcheck has been run via `cargo spellcheck --cfg=./spellcheck/era.cfg --code 1`.
+- [ ] Spellcheck has been run via `cargo spellcheck --cfg=./spellcheck/era.cfg --code 1` & `cspell docs/\*_/_.md
```

Comment:
let's add it to `zk` tool

Reply:
@Deniallugo added.

Reply:
Please update the template and github actions correspondingly